.github/workflows/test.yml

```diff
@@ -15,6 +15,13 @@ jobs:
           target: x86_64-unknown-linux-musl
           toolchain: nightly
           override: true
+      - name: "SEV: Verify certificate chain"
+        if: ${{ matrix.backend.name == 'sev' }}
+        run: |
+          ls -al /var/cache /var/cache/amd-sev /var/cache/amd-sev/chain
+          stat /var/cache/amd-sev/chain
+          cargo install sevctl
```

Comment:
`sevctl` isn't pushed to crates.io. You'll have to install via a git revision.

```console
cargo install --git https://github.com/enarx/sevctl
```

Can use the `--rev` switch to pin a certain commit to install from.